packages/site/src/content/docs/rules/ts/objectCalls.mdx

```diff
@@ -0,0 +1,79 @@
+---
+description: "Use object literal notation {} or Object.create instead of calling Object as a constructor."
+title: "objectCalls"
+---
+
+import { Tabs, TabItem } from "@astrojs/starlight/components";
+import { RuleEquivalents } from "~/components/RuleEquivalents";
+import RuleSummary from "~/components/RuleSummary.astro";
+
+<RuleSummary plugin="ts" rule="objectCalls" />
+
+Calling `Object` as a constructor with `new Object()` is unnecessarily verbose and less idiomatic than using object literal syntax.
+Object literal notation `{}` is the preferred and more concise way to create plain objects in JavaScript and TypeScript.
+
+## Examples
+
+<Tabs>
+<TabItem label="❌ Incorrect">
+
+```ts
+const emptyObject = new Object();
+```
+
+```ts
+const config = new Object({ key: "value" });
+```
+
+```ts
+function createEmpty() {
+	return new Object();
+}
+```
+
+</TabItem>
+<TabItem label="✅ Correct">
+
+```ts
+const emptyObject = {};
+```
+
+```ts
+const config = { key: "value" };
+```
+
+```ts
+function createEmpty() {
+	return {};
+}
+```
+
+```ts
+const objectWithoutPrototype = Object.create(null);
+```
+
+```ts
+const copy = Object.assign({}, source);
+```
+
+</TabItem>
+</Tabs>
+
+## Options
+
+This rule is not configurable.
+
+## When Not To Use It
+
+If you have a very large codebase with extensive use of `new Object()`, migrating all instances may be time-consuming.
+However, the migration is straightforward and the benefits of using object literals are significant.
```

Comment:
No, this isn't good. Instead go with two reasons:

* If you have an existing style that'll e difficult to migrate away from, such as for old consistency preferences
* If you target a legacy runtime with odd behavioral quirks